Gopkg.toml

```diff
@@ -67,9 +68,9 @@
   name = "github.com/kelseyhightower/envconfig"
   version = "1.3.0"
 
-[[constraint]]
-  branch = "master"
+[[constraint]] 
   name = "github.com/mitchellh/mapstructure"
+  branch = "master"
```

Comment:
what about pinning a version? if I am not mistaken, we had trouble one with mapstructure not being versioned

Reply:
They don't use semver, which makes this difficult, we will have to pin to a review (which is not good), but we could do it